Traceback (most recent call last):
  File "main.py", line 352, in <module>
    main()
  File "main.py", line 297, in main
    model.train(train_data, config, val_func=val_model)
  File "/home/jupyter/papers/regressGAN/replication/CGAN/models/cganmse.py", line 166, in train
    epoch_fooling.append(batch_fooling.item())
AttributeError: 'int' object has no attribute 'item'
